spec/services/zendesk_statistics_service_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+RSpec.describe ZendeskStatisticsService, type: :model do
```

Comment:
I think it would be useful if we had some more tests here that cover the failure cases. I'm particlarly interested in:

1. What happens when we can't communicate with the Zendesk API?
2. What happens if the format of titles, categories and tags aren't what you expect?

Reply:
Does this need to as much effort into this feature? It will only be used a handful of times by support agents/us so if it doesn't work as expected we can always inspect and fix it if need be.

Reply:
I still think some failure cases would be useful here. This is particularly important when making external API calls, and we've seen connectivity failures to Zendesk on the support form.

I noticed that the client configuration has `config.retry = true` — what does this mean? How many times will it retry?